client/post-editor/media-modal/detail/detail-fields.jsx

```diff
@@ -235,6 +241,38 @@ class EditorMediaModalDetailFields extends Component {
 		);
 	};
 
+	renderAllowDownloadOption = () => {
+		// Make sure this is actually a VideoPress video
+		const videopressGuid = this.getItemValue( 'videopress_guid' );
+		if ( ! videopressGuid ) {
+			return;
+		}
+
+		const allowDownloadKey = 'allow_download';
+		let allowDownload = this.getItemValue( allowDownloadKey );
+		if ( undefined === allowDownload ) {
+			allowDownload = 0;
+		}
+
+		return (
+			<EditorMediaModalFieldset legend={ this.props.translate( 'Download' ) }>
+				<FormLabel>
+					<FormCheckbox
+						id={ allowDownloadKey }
+						name={ allowDownloadKey }
+						checked={ allowDownload === '1' }
+						onChange={ this.handleAllowDownloadOption }
+					/>
+					<span>
+						{ this.props.translate(
+							'Display download option and allow viewers to download this video'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **43** times:
`translate( 'Display share menu and allow viewers to embed or download this video' )` *ES Score: 7*

Reply:
This is a new string